pom.xml

```diff
@@ -7,9 +7,9 @@
     </parent>
 
     <modelVersion>4.0.0</modelVersion>
-    <groupId>org.coursera</groupId>
+    <groupId>pro.streem</groupId>
     <artifactId>metrics-datadog-parent</artifactId>
-    <version>2.0.0-RC1</version>
+    <version>2.0.1</version>
     <packaging>pom</packaging>
     <name>Datadog Metrics Parent</name>
     <url>https://github.com/coursera/metrics-datadog</url>
```

Comment:
🔧 You should update the URL here and the various URLs below as well. Also the `developer` section should be updated.